### DISCUSSION TITLE
Installed a fresh copy of forge again... and it broke again

### DISCUSSION THREAD

**Discussion by wzwow** [lllyasviel/stable-diffusion-webui-forge#1011] [General]
I Installed a fresh copy of forge again... to try flux... it work for a night... and next day it wont start up... due to installing extension ?? 

cmd text here-
![image](https://github.com/user-attachments/assets/a6f2bc32-da94-469e-8547-c4b5fd8df4a9)

**Comment by DenOfEquity** [comment]
Miaoshou assistant extension is not currently compatible with new Forge.
Seems like it requires a package which has a dependency on an old version of another package, which then breaks Gradio.
See: https://github.com/miaoshouai/miaoshouai-assistant/issues/76

**Comment by ishadowx27** [comment]
remove all extensions and anything written by you on the webui-user.bat file. No xformers calls, nothing.

**Reply by wzwow** [reply]
seem like many extension not compatible with the new Gradio version... installed the aspect ratio thingy also not working.... seem like anything that has an ui isnt going to work....  

anyway i reinstall another fresh copy without anything extension gonna live without it for the moment...
